src/pretix/control/views/event.py

```diff
@@ -400,6 +405,78 @@ def post(self, request, *args, **kwargs):
             return self.get(request)
 
 
+class MailSettingsPreview(EventPermissionRequiredMixin, View):
+    permission = 'can_change_settings'
+
+    # return the origin text if key is missing in dict
+    class SafeDict(dict):
+        def __missing__(self, key):
+            return '{' + key + '}'
+
+    @cached_property
+    def items(self):
+        return {
+            'mail_text_order_placed': ['total', 'currency', 'date', 'invoice_company',
+                                       'event', 'paymentinfo', 'url', 'invoice_name'],
+            'mail_text_order_paid': ['event', 'url', 'invoice_name', 'invoice_company', 'payment_info'],
+            'mail_text_order_free': ['event', 'url', 'invoice_name', 'invoice_company'],
+            'mail_text_resend_link': ['event', 'url', 'invoice_name', 'invoice_company'],
+            'mail_text_resend_all_links': ['event', 'orders'],
+            'mail_text_order_changed': ['event', 'url', 'invoice_name', 'invoice_company'],
+            'mail_text_order_expire_warning': ['event', 'url', 'expire_date', 'invoice_name', 'invoice_company'],
+            'mail_text_waiting_list': ['event', 'url', 'product', 'hours', 'code']
+        }
+
+    @cached_property
+    def dummy_data(self):
+        return {
+            'event': self.request.event.name,
+            'total': 100,
+            'currency': self.request.event.currency,
+            'date': date_format(now() + timedelta(days=7), 'SHORT_DATE_FORMAT'),
+            'paymentinfo': _('Please transfer money to this bank account: BANK-ACCOUNT-0001'),
+            'payment_info': _('Please transfer money to this bank account: BANK-ACCOUNT-0001'),
+            'url': build_absolute_uri('presale:event.index', kwargs={
+                'event': self.request.event.slug,
+                'organizer': self.request.event.organizer
+            }),
+            'invoice_name': _('Invoice for {}.').format(self.request.event.name),
+            'invoice_company': self.request.event.organizer.name,
+            'orders': 'ORDER-0001\nORDER-0002\nORDER-0003\nORDER-0004',
+            'expire_date': date_format(now() + timedelta(days=15), 'SHORT_DATE_FORMAT'),
+            'product': _('Admission Ticket'),
+            'hours': 12,
```

Comment:
Although it sounds a little bit naive, I would like to ask a question regarding this setting.
I couldn't find the corresponding key-value pair in `pretixbase_eventsetting` table for `self.request.event.settings.waiting_list_hours`. How do I know where this setting is stored?
Would you mind giving me the direction?